src/reanimated2/animation/decay.ts

```diff
@@ -46,7 +46,7 @@ export function withDecay(
     'worklet';
     const config: DefaultDecayConfig = {
       deceleration: 0.998,
-      velocityFactor: Platform.OS !== 'web' ? 1 : 1000,
+      velocityFactor: !isWeb() ? 1 : 1000,
```

Comment:
Does it work? I should throw an exception because the `isWeb` function doesn't worklet.

Reply:
not fully tested, but AFAIK, it seems hostFunction is also runnable in worklet functions?